### ISSUE TITLE
[EN][NPC][Howling Fjord] It Goes to 11...

### ISSUE THREAD

**Issue by Norfik** [Atlantiss/BugTracker#1667]
http://www.wowhead.com/quest=11421/it-goes-to-11

Part with "Dragonflayer Defender slain (8)" is bugged. When we launch harpoons on buildings mobs are appearing but they don't fly to us and they are to far away to see them (server range of view in server settings don't allow such big radius to see them).

The way it works on Blizz: https://www.youtube.com/watch?v=NGwvNoZo5PU
The way it works on Alantiss: https://www.youtube.com/watch?v=Dg7uomI54pg

**Comment by bgb** [comment]
Yes, its bugged but there is workaround - on Atlantiss when you get on the harpoon you can actually still move. So just go closer (for example use roof which is under you) and from that place You can shoot Draonflayer Defenders and finish quest.

**Comment by krauza666** [comment]
There isn't any harpoon :(

**Comment by Illusionist87** [comment]
I can't get onto any harpoon. I used the quest item but it doesn't work

**Comment by Luna-maria** [comment]
work around doesn't work, can't move the harpoon anywhere :/

**Comment by DuGalle** [comment]
broken totaly confirm

**Comment by Hallieth** [comment]
Yap, doesn't work. You can't even lure mobs any near the harpoons' range...:(

**Comment by aendry** [comment]
still

**Comment by cyberknight1993** [comment]
Spears shot at buildings does not give you progress, building don't get destroyed. Dragon riders spawn but don't attack you, but you can try to kill them in air (there is water underneath so no death). Faction: Alliance.

**Comment by aendry** [comment]
@cyberknight1993 try my workaround to do it.

**Comment by aequitas54** [comment]
If you aim toward the back of the roof of the buildings, you will hit the buildings. It seems that the Defenders spawn in allocated locations in the sky. If you keep hitting the buildings in the correct spot then the one spawn point that is shootable will eventually spawn. Rinse Repeat till done
